tools/jira_client/jira_api.py

```diff
@@ -0,0 +1,34 @@
+from jira.exceptions import JIRAError
+import retry
+
+from .consts import CLOSED_TRANSITION_ID
+
+
+class JiraAPI:
+    def __init__(self, jira_client, logger) -> None:
+        self._jira_client = jira_client
+        self._logger = logger
+
+    def link_issue_to_root_issue(self, issue, root_issue):
+        self._logger.info("Linking issue key=%s to root issue key=%s", issue.key, root_issue.key)
+        self._jira_client.create_issue_link("relates to", issue.key, root_issue.key)
+
+    def close_issue(self, issue):
+        self._logger.info("Was requested to close issue %s ignoring for now", issue)
```

Comment:
sentence a bit weird, maybe just
```suggestion
        self._logger.info("Closing issue %s", issue)
```